### DISCUSSION TITLE
node exited with code -1073740791, whats is this error code?

### DISCUSSION THREAD

**Discussion by MuhamedSalihSeyedIbrahim** [nodejs/help#4868] [Q&A]
**node exited with code -_1073740791_**
**Need to know about this error code.**

**Description about our api:**
we have an api which do a simple post req.
onload it breaks with the exception.

_additional info:_
windows server.
nodejs 12x
axios 0.21x

**Comment by J-Human** [comment]
Maybe it would be possible to also include a sample code, and a media (if possible)?

**Reply by MuhamedSalihSeyedIbrahim** [reply]
I can't expose the code but what I can give is, as follows:
1. it a high load dealing azure serverless function 
2. language NodeJS
3. I receive a large JSON as a request and after manipulation, it will be ingested to some DB

want to know the general cause and if multiple a list may help?

**Reply by J-Human** [reply]
I don't have experience with Azure functions, and I've never heard of any negative exit codes, but perhaps it could be one of the following:
* Since you said you receive a large set of data, you could try compressing it, if you don't.
* It's possible the server-less function can't handle a huge request. 
* The database can handle these types of data and you're inserting them correctly, yes?

I can't "uncover" the very root of your problem, but I'm just presenting the least of what I know.

**Reply by MuhamedSalihSeyedIbrahim** [reply]
@J-Human, Yes we are inserting it properly but not sure we are doing any compression.

Just listing general causes might suffice my question.
Root cause analysis isn't needed.
The error from NodeJS is peculiar and  I couldn't find a doc for it. 
That why I just posted a Question in the discussion.

**Reply by J-Human** [reply]
> @J-Human, Yes we are inserting it properly but not sure we are doing any compression.
> 
> Just listing general causes might suffice my question.
> Root cause analysis isn't needed.
> The error from NodeJS is peculiar and I couldn't find a doc for it.
> That why I just posted a Question in the discussion.

Unfortunately, I honestly don't know about the error. But hopefully my answer earlier could give a tiny bit of information to the problem you're encountering.
